### PR TITLE
Add goftests 0.2.3

### PR DESCRIPTION
**Pull request by danring**

### CODE REVIEW

File: recipes/goftests/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - scikit-learn
+
+test:
+  imports:
+    - goftests
+
+about:
+  home: https://github.com/posterior/goftests
+  license: BSD-3-Clause
+  license_family: BSD
```
Comment (jakirkham) [inline recipes/goftests/meta.yaml]: Please ask or propose to upstream to add the license file to `MANIFEST.in` and link the issue/PR back here. Thanks.

Comment (danring) [inline recipes/goftests/meta.yaml]: Requested here: posterior/goftests#13

Comment (jakirkham) [inline recipes/goftests/meta.yaml]: Thanks. Please update this recipe to include `license_file: LICENSE.txt` on the next release that has that change.

**Comment by jakirkham** [comment]
Thanks @danring.

**Pull request merged**